test/helper.lua

```diff
@@ -230,4 +230,46 @@ function helpers.get_other_storage_bucket_id(cluster, bucket_id)
     ]], {bucket_id})
 end
 
+local function parse_tarantool_version(component)
```

Comment:
This code may be simpler if we're interested in just three components. See `crud/common/utils.lua`, `determine_enabled_features()` for example.

Reply:
Simplified now.